PlannTer/Modules/RoomModule/RoomDetailsView.swift

```diff
@@ -1,25 +1,34 @@
 import SwiftUI
 
 struct RoomDetailsView: View {
+    let title : String
     @StateObject private var controller = RoomDetailsController()
-    
+    @Environment(\.presentationMode) var presentationMode // To handle back navigation
     var body: some View {
-        NavigationView {
-            List(MockPlant.plants) { plant in
-                VStack(alignment: .leading) {
-                    Text(plant.name).font(.headline)
-                    
-                    HStack {
-                        Text("Water level: ")
-                        Text(String(plant.waterLevel))
+        VStack {
+            if /*@START_MENU_TOKEN@*/true/*@END_MENU_TOKEN@*/ {
```

Comment:
i may be missing something, but this one look suspicious at least
(did you insert some `if` statement by mistake or didn't finish editing it by any chance?)

Reply:
it is suspisious, chat made it, it's irrelevant, as well - room details still need to be replaced with actual content